webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -66,12 +68,18 @@ export class ContactsEffects {
         if (!selected) {
           return []; // return an empty stream if there is no selected contact
         }
+
         const refreshing = previousSelectedContact?.doc?._id === selected.id;
         this.globalActions.settingSelected(refreshing);
         this.globalActions.clearCancelCallback();
-        const options = { getChildPlaces: true };
-        const title = (selected.type && selected.type.name_key) || 'contact.profile';
+
+        const routeSnapshot = this.routeSnapshotService.get();
```

Comment:
My point of view:
The title shouldn't be set in effect & store, it's something that every page should be able to define freely. 
If in the future we add a new content component, 'contact new borns', then we need to add another condition here for the title to work properly, similar to what happened with 'contact decease'. Display components like 'navigation title' should ideally receive the data by input. Decreasing complexity in interaction between feature component, store, actions (unset and set title) and effects (usually complex).
Some other might be candidate for the same refactor (action bar?), something to think further and refactor in next iterations, I've added it in my to do list that I'll share after migration (fyi @craig-landry nice item consider later 🤩)

Reply:
I agree that the title should ideally be set in each component.